core/src/workflows/application.rs

```diff
@@ -23,7 +23,7 @@ pub async fn initialize(
             err
         ));
         let dna = dna.ok_or(HolochainError::DnaMissing)?;
-        context.log("dna/initialize: Initializing new chain from given DNA...");
+        context.log("initialize: Initializing new chain from given DNA...");
```

Comment:
I think this one still needs the "dna/" in front of the tag.

Reply:
Good catch! Thanks!